### PR TITLE
feat: run prettier on source code

### LINKED ISSUE #2201: run lint update after `button-dialog-fonts` branch being merged
#### Is this a bug, enhancement, or feature request?
enhancement.

#### Description
Run lint update after `button-dialog-fonts` branch being merged.

### PR DESCRIPTION
**Pull request by rengare**
#### Please provide a link to the associated issue.
Closes #2201 
#### Please provide a brief summary of this pull request.
Run prettier-cli with propper config  on source code
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done

### CODE REVIEW

**Comment by InnaAtanasova** [comment]
Maybe now that we released 0.16.0 is good time to merge this PR. Please rebase just incase before doing it :)

File: apps/docs/src/app/core/api-files.ts [hunk 1]
```diff
@@ -315,29 +222,9 @@ export const API_FILES = {
...
-    token: [
-        'TokenComponent',
-        'TokenizerComponent'
-    ],
-    tree: [
-        'TreeComponent',
-        'TreeChildComponent',
-        'TreeRowObjectModel'
-    ]
+    time: ['TimeComponent', 'TimeI18nLabels', 'TimeFormatParser', 'TimeFormatParserDefault', 'TimeObject'],
```
Comment (salarenko) [inline apps/docs/src/app/core/api-files.ts:225]: Can we prevent arrays from collapsing?

File: apps/docs/src/app/core/component-docs/action-bar/action-bar-header/action-bar-header.component.html [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <header>Action Bar</header>
-<description>The Action Bar is located at the top of the page and is used for Page title and Main Actions for the page.
+<description
+    >The Action Bar is located at the top of the page and is used for Page title and Main Actions for the page.
```
Comment (salarenko) [inline apps/docs/src/app/core/component-docs/action-bar/action-bar-header/action-bar-header.component.html:3]: This tag break is weird

File: libs/core/src/lib/link/link.component.ts [hunk 3]
```diff
@@ -50,7 +53,9 @@ export class LinkComponent implements OnChanges, OnInit, CssClassBuilder {
             this.disabled ? 'is-disabled' : '',
             this.inverted ? `fd-link--inverted` : '',
             this.class
-        ].filter(x => x !== '').join(' ');
+        ]
+            .filter((x) => x !== '')
```
Comment (salarenko) [inline libs/core/src/lib/link/link.component.ts:57]: I we could skip wrapping all arrow function arguments in braces

**Comment by droshev** [comment]
I don’t think it should be feat

**Comment by mikerodonnell89** [comment]
I can't remember if this has already been discussed.  But maybe we want to add a precommit hook?

**Comment by stefanoScalzo** [comment]
Need to resolve conflicts, and i think in general we should have a discussion on what we do and dont want as code refactoring @droshev @InnaAtanasova

**Comment by droshev** [comment]
addressed in #2410

**Pull request closed without merging**